src/modules/vmount/output.h

```diff
@@ -53,6 +53,7 @@ namespace vmount
 {
 
 struct OutputConfig {
+	uint32_t gimbal_as_uav_yaw;		/**< Use mixer output as UAV yaw */
```

Comment:
```suggestion
	int32_t gimbal_as_uav_yaw;		/**< Use mixer output as UAV yaw */
```

Given that's what the  param is as well.